AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -5245,6 +5245,7 @@ augenheilkunde.de#%#//scriptlet('set-cookie', 'CookieConsent', 'false')
 maxkuchnie.pl#%#//scriptlet("abort-current-inline-script", "$", "initRodoWindow();")
 discoelysium.com#%#(function() { try { if ("undefined" != typeof localStorage) { localStorage.setItem("cookieNotification", "closed") } } catch (ex) {} })();
 new.talktalk.co.uk#%#//scriptlet('set-cookie', 'TT_GDPR', 'true')
+exchange.uk.com#%#//scriptlet("set-cookie", "cookiePolicy", "true")
```

Comment:
what about these?
```
||exchange.uk.com/Public/Site/bundles/cookie?v=
||exchange.uk.com/Public/Login/Scripts/ibox.js
```

Reply:
I don't think it's ideal. The sites strictly says that is if cookies are not accepted, some site functionality will be limited.

Reply:
Fixed.